site/content/docs/08-link-options.md

```diff
@@ -2,32 +2,30 @@
 title: Link options
 ---
 
-### rel=prefetch
+### sapper:prefetch
 
 Sapper uses code splitting to break your app into small chunks (one per route), ensuring fast startup times.
 
 For *dynamic* routes, such as our `src/routes/blog/[slug].svelte` example, that's not enough. In order to render the blog post, we need to fetch the data for it, and we can't do that until we know what `slug` is. In the worst case, that could cause lag as the browser waits for the data to come back from the server.
 
-We can mitigate that by *prefetching* the data. Adding a `rel=prefetch` attribute to a link...
+We can mitigate that by *prefetching* the data. Adding a `sapper:prefetch` attribute to a link...
 
 ```html
-<a rel=prefetch href='blog/what-is-sapper'>What is Sapper?</a>
+<a href='blog/what-is-sapper' sapper:prefetch>What is Sapper?</a>
 ```
 
 ...will cause Sapper to run the page's `preload` function as soon as the user hovers over the link (on a desktop) or touches it (on mobile), rather than waiting for the `click` event to trigger navigation. Typically, this buys us an extra couple of hundred milliseconds, which is the difference between a user interface that feels laggy, and one that feels snappy.
 
-> `rel=prefetch` is a Sapper idiom, not a standard attribute for `<a>` elements
-
 <!-- TODO add a function to prefetch programmatically -->
 
-### rel=external
```

Comment:
> I've recently needed to navigate to a site in same-origin which wasn't a part of a Sapper app. Without rel=external it's not possible.

I would think we should just make the router fall through and treat a link as a normal link in the case that it can't find a corresponding route

Reply:
I'm confused about what specific case has an issue with treating `rel=external` in this way. I believe Sapper's behavior for the click interception is based on that of TJ's Page.js. In any case, IIRC Sapper already does bail on the interception if the link isn't to something that matches any route.

Reply:
@Conduitry my example:
- the main page is an Sapper app with link to articles at example.com
- articles aren't part of the Sapper app but are availble at example.com/category/article-title

Maybe both options should be available?

Reply:
But what happens if you don't put `rel=external`/`sapper:external`? I would expect it to still work in that case and that you shouldn't need this option

Reply:
If I don't mark "article" urls as `external` the Sapper app tries to load route because I use [...slug].svelte.
I understand that my example is highly

Reply:
Tries to load it and then what happens? You get a 404? What do you mean that use use `[...slug].svelte`?

Reply:
In routes I have `[...slug].svelte` to map all pathnames. So Sapper thinks there is a route for an article and after click loads template from `[...slug].svelte` with an error.
Then after page refresh VIP directs browser to a different app because articles are served by this app and not by Sapper.

I admit - it's a highly specific case.

Reply:
Maybe I should reduce this PR to a problem from #1401 and then open new issue about potential changes to `rel=external`?

Reply:
I still don't know what that has to do with making `rel=external` not have the normal page reload nav behavior. The idea is that `target` or `download` or `rel=external` will make it a non SPA link even if it matches some route.

Reply:
@Conduitry True, I fixated on my case and shouldn't remove functionality for `rel=external`.

I've made PR only for the `prefetch` https://github.com/sveltejs/sapper/pull/1566 and will close this one.